Drv/LinuxI2CDriver/LinuxI2CDriverComponentImpl.cpp

```diff
@@ -0,0 +1,90 @@
+// ======================================================================
+// \title  LinuxI2CDriverComponentImpl.cpp
+// \author aaron
+// \brief  cpp file for LinuxI2CDriver component implementation class
+//
+// \copyright
+// Copyright 2009-2015, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
+// ======================================================================
+
+
+#include <Drv/LinuxI2CDriver/LinuxI2CDriverComponentImpl.hpp>
+#include "Fw/Types/BasicTypes.hpp"
+
+#define I2C_FILE (char*)"/dev/i2c-1"
+
+int init_address(int addr) {
+  int fd = open(I2C_FILE, O_CLOEXEC | O_RDWR);
+  FW_ASSERT(fd >= 0);
+  int ioctlRes = ioctl(fd, I2C_SLAVE, addr);
```

Comment:
You use raw `int` types here?  Perhaps these should be made into `NATIVE_INT_TYPES`?

Reply:
Do you mean uint_8 or one of the NATIVE_INT_TYPES defined by F Prime? If the latter, what's the syntax to use it?

Reply:
https://github.com/nasa/fprime/blob/master/Fw/Types/BasicTypes.hpp defines the types, but here is a quick rundown.

```
U<bit width> defines unsigned int of that size. i.e. U8, U16, U32, and U64
I<bit width> defines signed int of that size. i.e. I8, I16, I32, and I64
F32, F64 are float and double.
```